families/track_and_trade/client/src/views/add_fish_form.js

```diff
@@ -19,70 +19,21 @@ const m = require('mithril')
 
 const payloads = require('../services/payloads')
 const transactions = require('../services/transactions')
+const {MultiSelect} = require('../components/forms')
+
+const PRECISION = payloads.FLOAT_PRECISION
```

Comment:
Looks like there are issues parsing user inputs into millionths and back again. 

This is what the output looks like when I enter a value of `1` for lat, long, and temp:
<img width="739" alt="screen shot 2017-09-27 at 10 51 32 am" src="https://user-images.githubusercontent.com/8889580/30923588-e4cf27b2-a371-11e7-863d-ec77906aa56b.png">

Reply:
Checking the database, it looks like these values are being submitted to the blockchain just as `1` (should be 1,000,000).

Similarly, inputting `3.14`, just gets submitted as `3` (should be 3,140,000).

Reply:
Pushed a commit to correct this.